timemachine/cpp/src/nonbonded_interaction_group.cu

```diff
@@ -119,13 +119,7 @@ NonbondedInteractionGroup<RealType>::NonbondedInteractionGroup(
 
     // estimate size needed to do radix sorting, this can use uninitialized data.
     cub::DeviceRadixSort::SortPairs(
-        d_sort_storage_,
-        d_sort_storage_bytes_,
-        d_sort_keys_in_,
-        d_sort_keys_out_,
-        d_sort_vals_in_,
-        d_perm_,
-        std::max(NC_, NR_));
+        d_sort_storage_, d_sort_storage_bytes_, d_sort_keys_in_, d_sort_keys_out_, d_sort_vals_in_, d_perm_, N_);
```

Comment:
@badisa I also changed this to use N_, otherwise this would overflow w/local MD when initialized only with the single dummy row/col idxs.